lib/kitchen/driver/gce_version.rb

```diff
@@ -18,6 +18,6 @@
 
 module Kitchen
   module Driver
-    GCE_VERSION = "1.2.0".freeze
+    GCE_VERSION = "1.2.1".freeze
```

Comment:
@zbikmarc Can you revert this change? We'll gather one or more fixes on master and then do a version bump as part of a release.

Reply:
Done